app/assets/stylesheets/local/index.scss

```diff
@@ -1,9 +1,147 @@
-.range_limit {
```

Comment:
Is this stylesheet really just for the home page?  Can we call it `home_page.scss` or something instead of `index.scss`?  There are lots of 'index' pages.